### ISSUE TITLE
Next.js next/image blocked by Wipr on Safari

### ISSUE THREAD

**Issue by galkowskit** [vercel/next.js#19215]
# Bug report

## Describe the bug
next/image is completely blocked by Wipr extension on Safari. https://twitter.com/tkg_codes/status/1328085125716971523 
While I understand the reasoning for treating this as not an issue with next/image - I think it will force users away from using next/image to not risk getting their actual content being hidden. At least some research could be done to make sure next/image is not blocked by content blockers.

## To Reproduce
Steps to reproduce the behavior, please provide code snippets or a repository:

1. Create new Next.js app.
2. Add any image.
3. Install Wipr extension in Safari.
4. Open the page with that browser.

https://tkg.codes/about <- site on which I saw it the first time.
https://github.com/galkowskit/next-image-issue <- sample repository that has the issue.
https://image-component.nextjs.gallery <- fails here as well.

## Expected behavior
Images should be visible.

## Screenshots
https://streamable.com/us0t91

## System information

- OS: macOS Big Sur 11.0.1 (20B29)
- Browser: Safari 14.0.1 (16610.2.11.51.8)
- Version of Next.js: 10.0.1
- Version of Node.js: 14.10.1

## Additional context
This error doesn't happen without Wipr. It was reported here (bottom most comment) https://github.com/vercel/next.js/issues/18681 and https://github.com/vercel/next.js/issues/19038.

**Comment by Timer** [comment]
We'll see if the author of Wipr can tell us why the plugin is incorrectly blocking our image component:
https://twitter.com/timer150/status/1328400951779209216

Happy to fix it if they can explain.

**Comment by Timer** [comment]
Wipr has fixed their blocklist:
https://twitter.com/mipstian/status/1328747255407448064

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
